client/app/states/services/details/details.state.js

```diff
@@ -87,12 +87,12 @@
     }
 
     function setInitialVars(vm) {
-      vm.showRemoveService = $state.actionFeatures.service_delete.show;
-      vm.showRetireService = $state.actionFeatures.service_retire_now.show;
-      vm.showScheduleRetirementService = $state.actionFeatures.service_retire.show;
-      vm.showEditService = $state.actionFeatures.service_edit.show;
-      vm.showReconfigureService = $state.actionFeatures.service_reconfigure.show;
-      vm.showSetOwnership = $state.actionFeatures.service_ownership.show;
+      vm.showRemoveService = $state.actionFeatures.serviceDelete.show;
```

Comment:
it was originally snake, against best practice (eslint and miq guide), just following through on the implementation